forge-gui/res/cardsfolder/upcoming/w_3_3_bird_flying_jump.txt

```diff
@@ -0,0 +1,10 @@
+Name:Bird Token
```

Comment:
The file name of this token should match that in the edition file. Please check editions for the correct names.

w_3_3_bird_flying_attacks